A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Autopilot.xml

```diff
@@ -0,0 +1,165 @@
+<ModelBehaviors>
+
+    <!--
+    Template for the AP1/AP2 master button on the FCU.
+
+    Should probably be replaced by a generic template at some point.
+    -->
+    <Template Name="FBW_Airbus_Autopilot_Template">
+        <DefaultTemplateParameters>
+            <DEFAULT_NAME>Autopilot</DEFAULT_NAME>
+            <HELPID>HELPID_AUTOPILOT_MASTER</HELPID>
+            <TOOLTIPID>%((A:AUTOPILOT MASTER, bool))%{if}TT:COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_AP_ON%{else}TT:COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_AP_OFF%{end}</TOOLTIPID>
+            <AP_COUNT>1</AP_COUNT>
+            <ID>1</ID>
+            <ON_AP_MASTER_SWITCH></ON_AP_MASTER_SWITCH>
+        </DefaultTemplateParameters>
+        <OverrideTemplateParameters>
+            <Switch Param="TYPE">
+                <Case Value="EXCLUSIVE_AP">
+                    <Switch Param="AP_COUNT">
+                        <Case Value="2">
+                            <RESET_CODE>
+                                1 #ID# != if{ 0 (&gt;L:XMLVAR_Autopilot_1_Status) }
+                                2 #ID# != if{ 0 (&gt;L:XMLVAR_Autopilot_2_Status) }
+                            </RESET_CODE>
+                        </Case>
+                    </Switch>
```

Comment:
We can remove the surrounding `Switch` and `Case` tags, since there is only ever 2 APs on the A320.

Reply:
this is more or less a copy of the existing asobo template, except the LEFT_SINGLE_CODE Part. As you know this RNP/Template stuff is still my week point. I tried to remove everything were i was sure i can :P ... So in this case ... do i really only remove the 4 lines and its fine? Because if i just can delete them and the syntax remains intact, i can do this aswell for all other switch/case in this template (there are about 2-3 others)

Reply:
can you please review if the new commit i fine ? i tested inflight and it seems i did not brake something at least :P

Reply:
LGTM